src/components/forums/topic/GiveAward.tsx

```diff
@@ -118,17 +161,20 @@ export function GiveAward(props: GiveAwardProps) {
                 name="award"
```

Comment:
can you replace this component with the Input.tsx component as well in `components/common`